lib/ts3admin.class.php

```diff
@@ -3263,11 +3263,12 @@ private function loadQueryData()
   * @param		boolean	$virtual	set true to add -virtual param [optional]
   * @return     array success
   */
-	function selectServer($value, $type = 'port', $virtual = false) { 
+	function selectServer($value, $type = 'port', $virtual = false, $name = null) { 
+		if($name == null && strlen($name) >=3 ){ $name = ''; }else{ $name = " client_nickname=".$name.""; }
```

Comment:
if `$name` equals `null` then what result would be `strlen($name)`  :face_with_head_bandage: 
rethink this line

Reply:
take a look at https://github.com/par0noid/ts3admin.class/blob/master/lib/ts3admin.class.php#L4253
and rethink

Reply:
You don't need to check if the value is Null cause a strlen of Null is 0.